ospfd/ospf_vty.c

```diff
@@ -3977,11 +4239,27 @@ DEFUN (show_ip_ospf_neighbor_int,
 {
   struct ospf *ospf;
   u_char uj = use_json(argc, argv);
+  struct listnode *node = NULL;
+  int ret = CMD_SUCCESS;
+  struct interface *ifp = NULL;
+  int idx_if = 4;
 
-  if ((ospf = ospf_lookup()) == NULL || !ospf->oi_running)
-    return CMD_SUCCESS;
+  if (!uj)
+    show_ip_ospf_neighbour_header (vty);
 
-  return show_ip_ospf_neighbor_int_common(vty, ospf, 0, argv, uj);
+  ifp = if_lookup_by_name_all_vrf (argv[idx_if]->arg);
+  for (ALL_LIST_ELEMENTS_RO (om->ospf, node, ospf))
+    {
+      if (!ospf->oi_running)
+        continue;
+      if (IS_DEBUG_OSPF_TRACE)
+        zlog_debug ("%s: ifp->vrf_id %d ospf->vrf_id %d", __PRETTY_FUNCTION__,
```

Comment:
This debug gives us no clue as to what command is being run.  What actual debugability does this provide us?